speech-to-text/v1.ts

```diff
@@ -279,6 +279,7 @@ namespace SpeechToTextV1 {
     splitTranscriptAtPhraseEnd?: boolean;
     speechDetectorSensitivity?: number;
     backgroundAudioSuppression?: number;
+    lowLatency?: boolean;
```

Comment:
This needs to be accompanied by changes in `lib/recognize-stream.ts` in order for the new parameter to actually be supported

Reply:
Thank you, I did not realize this